.gitlab-ci.yml

```diff
@@ -1148,6 +1148,9 @@ plugin:ci-fiat_parsers:
 plugin:ci-lean_importer:
   extends: .ci-template
 
+plugin:ci-lean4_importer:
+  extends: .ci-template
```

Comment:
It may be nicer to use ci-template-flambda if you don't want to have the same minimum ocaml version as Rocq (ci-template is 4.09, -flambda is 4.14).